USAGE_android.md

```diff
@@ -919,6 +919,8 @@ optional arguments:
               Enables dumping of all image sub resources (mip map levels and array layers)
   --dump-resources-dump-raw-images
               When enabled all image resources will be dumped verbatim as raw bin files.
+  --dump-resources-dump-separate-alpha
```

Comment:
nit, I don't mind much using this option, but for other gfxr users, I wonder if it should not be the default, even if we do need the ability not do dump it separately.

Reply:
An option might make more sense if gfxr would emit the alpha for all png's, say also for screenshots if this option is present. And not emit them otherwise. But as I said, I don't mind using the option much, just with its current state to only apply to the dump resources api's, it's not the default I would have picked :-).

Reply:
Other clients of the dump resources API don't have a problem with combined alpha.  I think default combined is fine.